act/qc/__init__.py

```diff
@@ -25,9 +25,11 @@
     ],
     submod_attrs={
         'arm': ['add_dqr_to_qc'],
-        'qcfilter': ['QCFilter'],
```

Comment:
do we still need to specify QCFilter and QCtests

Reply:
@zssherman  good question.  I took out QCTests from the modules but we need to keep QCFilter in there because there are 3 functions outside the class that we need to load up that we are not currently doing so.